ext/build_defaults.yaml

```diff
@@ -21,6 +21,11 @@ build_msi:
   facter:
     ref: 'refs/tags/2.3.0'
     repo: 'git://github.com/puppetlabs/facter.git'
+  cfacter:
+    archive:
+      x86: 'cfacter-0.2.0-377-g4fa6175-x86.zip'
+      x64: 'cfacter-0.2.0-377-g4fa6175-x64.zip'
+    path: 'http://builds.puppetlabs.lan/cfacter/4fa61758e3f4f2bf909da5eb5007f197c04707f5/artifacts/windows/'
```

Comment:
I'm expecting there'll be churn. I'd suggest waiting until the cfacter release next week CFACT-182